### ISSUE TITLE
Inconsistency with scrolling on status_bar

### ISSUE THREAD

**Issue by freed00m** [tobi-wan-kenobi/bumblebee-status#188]
Why does scrolling over modules cycles through workspaces?

I mean when I hover over mudule like ram or cpu and scroll, I can get from w1 -> w2 -> w9 -> w1 whith  just scrolling up. However if my pointer is not over the module but on the bar itself I get stopped at w9.

I went through documentation on both i3 and bumblebee and I didn't catch any additional setting that changes the behaviour. 
 I don't want to cycle through workspaces just so is it possible to prevent bumblebee from cycling on scroll?

**Comment by tobi-wan-kenobi** [comment]
Hi,

well, it's a feature I initially found in `awesomewm` and really missed in i3, until I added it to my bar. The different behavious is annoying, you are right, I didn't notice that (my bar behaves differently than the i3bar itself).

I will have a look at that during the weekend, and also create an option to disable the scrolling.

Thanks for the input!

**Comment by tobi-wan-kenobi** [comment]
Hi,

I've now added an option to completely disable scrolling on the bar.

Regarding the inconsistency (i3bar itself stops at last workspace, while bumblebee-status wraps): I've looked through the i3 documentation, but was unable to find a command that does that. I think I'd have to manually query all the workspaces, find the active output, and only rotate if it's not the first/last workspace on the currently-active output.

Given that this sounds like something where I'd introduce tons of bugs, I think I'd rather postpone that, if you don't mind.

I think your major issue has been solved with the previous commit, right?

Cheers!
